src/types/cardano.ts

```diff
@@ -0,0 +1,169 @@
+import BigNumber from 'bignumber.js'
+
+// Wallet Candidates
+
+export type TransactionDirection = 'SENT' | 'RECEIVED' | 'SELF' | 'MULTI'
+export type TransactionStatus = 'SUCCESSFUL' | 'PENDING' | 'FAILED'
+export type TransactionAssurance = 'PENDING' | 'FAILED' | 'LOW' | 'MEDIUM' | 'HIGH'
+
+export type Block = {
```

Comment:
my guess is, all blockchains will have blocks with heights and hashes, but not all will have epochs or slots
so, maybe something like:
```ts
namespace Yoroi {
  type Block = {
    height: number, 
    hash: string
  }
}
```
and 

```ts
namespace Cardano {
  type Block = {
    height: number, 
    hash: string, 
    epoch: number, 
    slot: number
  }
}
```